### DISCUSSION TITLE
Question about addon versioning

### DISCUSSION THREAD

**Discussion by goldenstein64** [LuaLS/LLS-Addons#156] [Q&A] [answered]
I've been getting PRs for penlight since it was updated to 1.14.0; would it be better to upload a new addon with the version updated or simply update the old addon?

**Comment by carsakiller** [comment, marked as answer]
I think it depends on your specific use-case. If the changes are not breaking, I would say same addon. I only really see a need for a new addon for “2.0s” or large refactors.

In the long term, I would like for there to be better support for versioning with LLS-Addons, which would probably also support updating just one addon or not use Git at all.

**Reply by carsakiller** [reply]
@goldenstein64 I have just opened a PR on the extension that adds a rough version selector using git tags.

This should allow users to select "Latest" to stay on the latest commit or a specific tag to jump to.
